From: Sophie Brun <sophie@freexian.com>
Date: Wed, 19 Sep 2018 16:34:06 +0800
Subject: Add missing "include"

Add #include <string.h> to fix the warnings: "implicit
declaration of function 'memcpy'" and "implicit declaration of function
'memset'"
Last-Update: 2016-03-11
---
 common/lib/r_assoc.c    | 1 +
 common/lib/r_bitfield.c | 1 +
 common/lib/r_data.c     | 1 +
 3 files changed, 3 insertions(+)

diff --git a/common/lib/r_assoc.c b/common/lib/r_assoc.c
index 2fee9af..a98534a 100644
--- a/common/lib/r_assoc.c
+++ b/common/lib/r_assoc.c
@@ -56,6 +56,7 @@
 
 static char *RCSSTRING="$Id: r_assoc.c,v 1.4 2001/12/24 06:06:26 ekr Exp $";
 
+#include <string.h>
 #include <r_common.h>
 #include "r_assoc.h"
 
diff --git a/common/lib/r_bitfield.c b/common/lib/r_bitfield.c
index 5ca2839..522da81 100644
--- a/common/lib/r_bitfield.c
+++ b/common/lib/r_bitfield.c
@@ -10,6 +10,7 @@
 
 static char *RCSSTRING="$Id: r_bitfield.c,v 1.3 2001/12/24 06:06:26 ekr Exp $";
 
+#include <string.h>
 #include <r_common.h>
 #include "r_bitfield.h"
 
diff --git a/common/lib/r_data.c b/common/lib/r_data.c
index d3e136e..f15767a 100644
--- a/common/lib/r_data.c
+++ b/common/lib/r_data.c
@@ -46,6 +46,7 @@
 
 static char *RCSSTRING="$Id: r_data.c,v 1.3 2001/07/20 23:33:15 ekr Exp $";
 
+#include <string.h>
 #include <r_common.h>
 #include <r_data.h>
 
